client/my-sites/subscribers/main.tsx

```diff
@@ -1,19 +1,306 @@
+		subscription_id: 15,
+		display_name: 'María Guzmán',
+		email_address: 'mariaguzman@gmail.com',
+		status: 'inactive',
+		subscription: 'Free Tier',
+		openRate: '60%',
+		date_subscribed: '2022-04-01T00:00:00.000Z',
+		avatar: '',
+	},
+	{
+		user_id: 16,
+		subscription_id: 16,
+		display_name: 'Javier Salazar',
+		email_address: 'javisalazar@gmail.com',
+		status: 'active',
+		subscription: 'Annual (US$ 50)',
+		openRate: '100%',
+		date_subscribed: '2022-05-01T00:00:00.000Z',
+		avatar: '',
+	},
+	{
+		user_id: 17,
+		subscription_id: 17,
+		display_name: 'Laura Mendoza',
+		email_address: 'lauramendoza@gmail.com',
+		status: 'active',
+		subscription: 'Monthly (US$ 4.99)',
+		openRate: '85%',
+		date_subscribed: '2022-06-01T00:00:00.000Z',
+		avatar: '',
+	},
+	{
+		user_id: 18,
+		subscription_id: 18,
+		display_name: 'Pablo Aguirre',
+		email_address: 'pabloaguirre@gmail.com',
+		status: 'inactive',
+		subscription: 'Free Tier',
+		openRate: '80%',
+		date_subscribed: '2022-07-01T00:00:00.000Z',
+		avatar: '',
+	},
+];
 
 export const Subscribers = () => {
 	const isSubscribersPageEnabled = config.isEnabled( 'subscribers-page' );
+	const locale = useLocale();
+
+	const navigationItems: Item[] = [
+		{
+			label: 'Subscribers',
+			href: `/subscribers`,
+			helpBubble: (
+				<span>
+					{ translate(
+						'Add subscribers to your site and send them a free or paid {{link}}newsletter{{/link}}.',
+						{
+							components: {
+								link: (
+									<a
+										href="https://wordpress.com/support/launch-a-newsletter/#about-your-subscribers"
+										target="blank"
+									/>
+								),
+							},
+						}
+					) }
+				</span>
+			),
+		},
+	];
 
 	if ( ! isSubscribersPageEnabled ) {
 		return null;
 	}
 
 	return (
-		<Main>
+		<>
 			<DocumentHead title={ translate( 'Subscribers' ) } />
-			Subscribers
-		</Main>
+			<div className="main is-wide-layout">
+				<FixedNavigationHeader navigationItems={ navigationItems }></FixedNavigationHeader>
+				<div className="header-count">
+					<span className="title">{ translate( 'Total' ) }</span>{ ' ' }
+					<span className="subscriber-count">{ subscribers && subscribers.length }</span>
+				</div>
+				<table className="subscriber-list">
+					<tbody>
+						<tr className="table-header">
+							<th className="checkbox-column">
+								<FormCheckbox />
+							</th>
+							<th className="profile-column">Name</th>
+							<th className="subscription-type-column">Subscription type</th>
+							<th className="rate-column">Open Rate</th>
+							<th className="since-column">Since</th>
+							<th className="ellipsis-menu"></th>
+						</tr>
+						{ subscribers.map(
+							( {
+								subscription_id,
+								display_name,
+								email_address,
+								subscription,
+								openRate,
+								date_subscribed,
+								avatar,
+							} ) => (
+								<tr key={ subscription_id }>
+									<td className="checkbox-column">
+										<FormCheckbox />
+									</td>
```

Comment:
On the phase 1 we aren't including the checkbox, so maybe it would worth it to remove for now. h8tMpJlCqNFemEerU9owHI-fi-1880_61261